src/pool/srv_pool.c

```diff
@@ -143,6 +143,8 @@ struct pool_svc {
 	struct pool_svc_events	ps_events;
 	uint32_t		ps_global_version;
 	struct pool_svc_reconf	ps_reconf;
+	/* The global pool map version on all pool targets */
+	uint32_t		ps_global_map_version;
```

Comment:
Not a showstopper, but if you think it's a good idea and if you happen to update this patch any further:

To avoid confusion with ps_global_version (that is associated with metadata layout version), maybe this could instead be called "ps_dist_map_version"?

Similarly in rebuild_task_ult() maybe the local variable global_ver could be named differently (perhaps dist_ver or dist_map_ver?).

Reply:
yes, makes sense. will update if I need update the patch. thanks.